gometa_test.go

```diff
@@ -48,6 +49,7 @@ func TestGometa(t *testing.T) {
 <html>
 <head>
 <meta name="go-import" content="empty.com/test git ">
+<meta name="go-source" content="empty.com/test _ /tree/v2{/dir} /blob/v2{/dir}/{file}#L{line}">
```

Comment:
Still not sure what we should do in the ``go-source`` tag when ``rec.To`` is empty.

Reply:
What are the cases, where rec.To is empty?
Maybe not expose "go-source" at all?